ArjunaJTS/recovery/README.md

```diff
@@ -1,73 +1,79 @@
-JBoss, Home of Professional Open Source
```

Comment:
removed copyright?

Reply:
yes, it's intentional and I think it should be removed, see above